cvat-ui/src/components/profile-page/security-content.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { Col, Row } from 'antd/lib/grid';
+import Card from 'antd/lib/card';
+import Typography from 'antd/lib/typography';
+import Button from 'antd/lib/button';
+
+import { changePasswordAsync } from 'actions/auth-actions';
+import ChangePasswordForm, { ChangePasswordData } from './change-password-form';
+
+function SecurityContent(): JSX.Element {
+    const dispatch = useDispatch();
+    const [showForm, setShowForm] = useState(false);
+
+    const onChangePassword = (data: ChangePasswordData): void => {
+        dispatch(changePasswordAsync(data.oldPassword, data.newPassword1, data.newPassword2));
+        setShowForm(false);
```

Comment:
Should not we hide form only on success?

Reply:
Added onSuccess callback